core/scripts/chaincli/handler/keeper_deployer.go

```diff
@@ -0,0 +1,161 @@
+}
+
+type v12KeeperDeployer struct {
+	registry12.KeeperRegistryInterface
+}
+
+func (d *v12KeeperDeployer) SetKeepers(opts *bind.TransactOpts, _ []cmd.HTTPClient, keepers []common.Address, payees []common.Address) (*types.Transaction, error) {
+	return d.KeeperRegistryInterface.SetKeepers(opts, keepers, payees)
+}
+
+type v20KeeperDeployer struct {
+	registry20.KeeperRegistryInterface
+	cfg *config.Config
+}
+
+func (d *v20KeeperDeployer) RegisterUpkeep(opts *bind.TransactOpts, target common.Address, gasLimit uint32, admin common.Address, checkData []byte) (*types.Transaction, error) {
+	return d.KeeperRegistryInterface.RegisterUpkeep(opts, target, gasLimit, admin, checkData, nil)
+}
+
+func (d *v20KeeperDeployer) SetKeepers(opts *bind.TransactOpts, cls []cmd.HTTPClient, keepers []common.Address, _ []common.Address) (*types.Transaction, error) {
+	oracleIdentities := make([]ocr2config.OracleIdentityExtra, len(cls))
+	var wg sync.WaitGroup
+	for i, cl := range cls {
+		wg.Add(1)
+		go func(i int, cl cmd.HTTPClient) {
+			defer wg.Done()
+
+			ocr2Config, err := getNodeOCR2Config(cl)
+			if err != nil {
+				panic(err)
+				return
+			}
+
+			p2pKeyID, err := getP2PKeyID(cl)
+			if err != nil {
+				panic(err)
+				return
+			}
+
+			offchainPkBytes, err := hex.DecodeString(strings.TrimPrefix(ocr2Config.OffChainPublicKey, "ocr2off_evm_"))
+			if err != nil {
+				panic(fmt.Errorf("failed to decode %s: %v", ocr2Config.OffChainPublicKey, err))
+				return
+			}
+
+			offchainPkBytesFixed := [ed25519.PublicKeySize]byte{}
+			n := copy(offchainPkBytesFixed[:], offchainPkBytes)
+			if n != ed25519.PublicKeySize {
+				panic(fmt.Errorf("wrong num elements copied"))
+				return
+			}
+
+			configPkBytes, err := hex.DecodeString(strings.TrimPrefix(ocr2Config.ConfigPublicKey, "ocr2cfg_evm_"))
+			if err != nil {
+				panic(fmt.Errorf("failed to decode %s: %v", ocr2Config.ConfigPublicKey, err))
+				return
+			}
+
+			configPkBytesFixed := [ed25519.PublicKeySize]byte{}
+			n = copy(configPkBytesFixed[:], configPkBytes)
+			if n != ed25519.PublicKeySize {
+				panic(fmt.Errorf("wrong num elements copied"))
+				return
+			}
+
+			onchainPkBytes, err := hex.DecodeString(strings.TrimPrefix(ocr2Config.OnchainPublicKey, "ocr2on_evm_"))
+			if err != nil {
+				panic(fmt.Errorf("failed to decode %s: %v", ocr2Config.OnchainPublicKey, err))
+				return
+			}
+
+			oracleIdentities[i] = ocr2config.OracleIdentityExtra{
+				OracleIdentity: ocr2config.OracleIdentity{
+					OnchainPublicKey:  onchainPkBytes,
+					OffchainPublicKey: offchainPkBytesFixed,
+					PeerID:            p2pKeyID,
+					TransmitAccount:   ocr2types.Account(keepers[i].String()),
+				},
+				ConfigEncryptionPublicKey: configPkBytesFixed,
+			}
+		}(i, cl)
+	}
+	wg.Wait()
+
+	signers, transmitters, f, onchainConfig, offchainConfigVersion, offchainConfig, err := ocr2config.ContractSetConfigArgsForEthereumIntegrationTest(oracleIdentities, 1, uint64(1000))
```

Comment:
We'll need to refactor this function. Currently this adds on the median plugin config inside offChainConfig, need to change it for our plugin. cc @EasterTheBunny